assets/js/pages/Users/UsersPage.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { listUsers, deleteUser } from '@lib/api/users';
+import { format, parseISO } from 'date-fns';
+
+import { toast } from 'react-hot-toast';
+
+import Users from './Users';
+
+function UsersPage() {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [deleteUserId, setDeleteUserId] = useState(null);
+  const [userUpdateTrigger, setUserUpdateTrigger] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const [userData, setUserData] = useState([]);
+  const [error, setError] = useState(null);
+
+  const navigate = useNavigate();
+
+  const fetchUsers = () => {
+    setLoading(true);
+    listUsers()
+      .then((response) => {
+        setUserData(response.data);
+        setLoading(false);
+      })
+      .catch((_error) => {
+        setError('An error occurred during loading users');
+        setUserData([]);
+        setLoading(false);
+      });
+  };
+
+  const handleDeleteUser = (userId) => {
+    deleteUser(userId)
+      .then(() => {
+        setUserUpdateTrigger(true);
```

Comment:
Do we want to fetch users again here?
It is not simpler just removing the entry from the list.
Less operations...
And we know that it succedded.

Reply:
I think refetching the users after a delete it's a better choice, we don't have inconsistencies and the call is cheap, manipulating. It's better to have the data api immutable and display it

Just don't use the trigger and call the function directly